frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -1,21 +1,39 @@
 import React, { useEffect, useState } from "react";
-import { getSightings, Sighting } from "../../clients/apiClient";
+import {
+  getExternalApiSightings,
+  getSightings,
+  Sighting,
+  SightingFromExternalApi,
+} from "../../clients/apiClient";
 import { SightingCard } from "./SightingCard";
+import { SightingCardForExternalApi } from "./SightingCardForExternalApi";
 import "./BrowseSightings.scss";
 
 export const BrowseSightings: React.FC = () => {
   const [sightings, setSightings] = useState<Sighting[]>();
+  const [externalApiSightings, SightingFromExternalApi] =
+    useState<SightingFromExternalApi[]>();
 
   useEffect(() => {
     getSightings().then(setSightings);
+    getExternalApiSightings().then(SightingFromExternalApi);
   }, []);
 
-  if (sightings === undefined) {
+  if (sightings === undefined || externalApiSightings === undefined) {
     return <p>Loading</p>;
   }
 
   return (
     <>
+      <h1>Reported Sightings From External Api</h1>
+      <ul>
+        {externalApiSightings.map((apiSighting, index) => (
+          <SightingCardForExternalApi
+            apiSighting={apiSighting}
+            key={apiSighting.id}
+          />
+        ))}
+      </ul>
```

Comment:
This change will massively conflict with another ticket that Nick and Kostas are doing/have done that splits this component into 2, and you'll want to use that here